### PR TITLE
Fix/acceptance-vars

### LINKED ISSUE #70: make acceptance uses inconsistent env variables
as per the conversation in #66 ... we found that `make acceptance` uses different variables than the rest.

The rest of the make targets use:
> `JFROG_ACCESS_TOKEN` / `ARTIFACTORY_URL`

make acceptance uses:
> `ARTIFACTORY_ACCESS_TOKEN` / `JFROG_URL`

Should we change the test code to `ARTIFACTORY_URL` and `JFROG_ACCESS_TOKEN` to match the test, or change everything to JFROG_blah or ARTIFACTORY_blah. It does seem a bit strange to use ARTIFACTORY_ for one and JFROG_ for the other.

### PR DESCRIPTION
**Pull request by TJM**
This fixes #70. It will use the Makefile variables that we have used for everything else.

* make
(in another window)
* make artifactory
* make acceptance

### CODE REVIEW

**Review by alexhung — DISMISSED** [review]
You beat me to it! 😄

File: README.md [hunk 1]
```diff
@@ -263,6 +263,21 @@ username           v-jenkins-x4mohTA8
 
 ## Development
 
+### Local Development Prerequisites
```
[the remark this answers is not in the document]
Comment (TJM) [reply README.md:266]: sorry, I started this branch from my last branch, so I just pulled the *other* change out :)

The readme changes can go either way :)

**Comment by alexhung** [comment]
@TJM Probably best to merge #66 first, then rebase/update this branch.

**Review by alexhung — APPROVED** [review]

File: Makefile
Comment (alexhung) [inline Makefile:1]: Strange how this PR isn't picking up these changes from `master` already.

**Pull request merged**